web/apps/benchmarks-and-reports/src/app/(benchmarks-and-reports)/[version]/applications-benchmarks/[slug]/_components/applications-benchmarks-table-schema.ts

```diff
@@ -1,13 +1,28 @@
 import { z } from "zod";
 
-const applicationsBenchmarksTableSchema = z.object({
-  name: z.string(),
-  size: z.string(),
-  speed: z.string(),
-  total_duration: z.string(),
-  total_cycles: z.string(),
-  user_cycles: z.string(),
-  proof_bytes: z.string(),
-});
+const applicationsBenchmarksTableSchema = {
+  main: z.object({
+    name: z.string(),
+    size: z.string(),
+    speed: z.string(),
+    total_duration: z.string(),
+    total_cycles: z.string(),
+    user_cycles: z.string(),
+    proof_bytes: z.string(),
+  }),
+  "release-0.21": z.object({
+    job_name: z.string(),
+    job_size: z.string(),
+    exec_duration: z.string(),
+    proof_duration: z.string(),
+    total_duration: z.string(),
+    verify_duration: z.string(),
+    insn_cycles: z.string(),
+    prove_cycles: z.string(),
+    proof_bytes: z.string(),
+  }),
+};
```

Comment:
@flaub this defines 2 different schemas for the different data shapes between 0.21 and main

Reply:
Great, thank you!